src/pykiso/test_result/assert_step_report.py

```diff
@@ -476,3 +480,32 @@ def generate_step_report(
     # Write the output into the output file
     with output_file.open("w") as report_file:
         report_file.write(output_text)
+
+
+def add_retry_information_in_step_report(
```

Comment:
```suggestion
def add_retry_information(
```
It's already in the assert_step_report module ;)

Reply:
Done